Description: Upstream changes introduced in version 02.07.2010.08.24-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 aptosid-manual (02.07.2010.08.24-1) unstable; urgency=low
 .
   * bump standards version to 3.9.1, no changes necessary.
   * add a Breaks: sidux-manual-reader (<< 0.2), as old versions can't parse
     with the current menu structure of sidux-manual.
   * switch to compat level 8.
   * add debian/bootstrap.sh, to ease language additions/ removals.
   * fill in LL_LOCAL_LANGUAGE.
   * switch debian/watch to aptosid.
   * rename to aptosid, disable bg, da, hu and pl for the time being.
   * New upstream release.
 .
 The person named in the Author field signed this changelog entry.
Author: Stefan Lippers-Hollmann <s.l-h@gmx.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ aptosid-manual-02.07.2010.08.24/menu/aptosid-manual.desktop
@@ -0,0 +1,24 @@
+[Desktop Entry]
+Type=Application
+Exec=x-www-browser /usr/share/aptosid-manual/index.html
+Icon=aptosid-manual
+Name=aptosid manual
+Name[bg]=aptosid Manual
+Name[da]=aptosid Manual
+Name[de]=aptosid Handbuch
+Name[el]=Εγχειρίδιο aptosid
+Name[en_GB]=aptosid Manual
+Name[en_US]=aptosid Manual
+Name[es]=Manual de aptosid
+Name[fr]=Manuel de aptosid
+Name[hr]=aptosid Priručnik
+Name[hu]=aptosid Manual
+Name[it]=Manuale di aptosid
+Name[ja]=aptosid Manual
+Name[nl]=aptosid Handleiding
+Name[pl]=aptosid Manual
+Name[pt]=Manual do aptosid
+Name[pt_BR]=Manual do aptosid
+Name[ro]=aptosid Manual
+Name[ru]=aptosid руководство
+Categories=System;Documentation
\ No newline at end of file
